src/lib/components/support.svelte

```diff
@@ -44,7 +44,18 @@
                 </p>
             {/if}
         </div>
-        {#if $organization?.billingPlan === BillingPlan.STARTER}
+        {#if $organization?.billingPlan === BillingPlan.PRO || $organization?.billingPlan === BillingPlan.SCALE}
+            <Button
+                secondary
+                fullWidth
+                on:click={() => {
+                    show = false;
+                    $showSupportModal = false;
+                    wizard.start(SupportWizard);
+                }}>
+                <span class="text">Contact our Support Team</span>
+            </Button>
+        {:else}
```

Comment:
We should also handle the case where I click the "Get Premium support" when I'm on the account page and I do have an org. Right now, it shows the "Get premium support" button, but clicking does nothing (because `$organization` isn't set).

<img width="1337" alt="image" src="https://github.com/appwrite/console/assets/1477010/7197cbbe-62f8-4831-ac87-bf8398ee2174">